src/hardware/steering_actuator/src/node_steering_actuator.cpp

```diff
@@ -459,8 +459,8 @@ class SteeringActuator : public rclcpp::Node, public CanInterface {
                 double derivative_error = (error - this->prev_error) / elapsed_time_seconds;  // Grab derivative error
 
                 // left hand down is +, rhd is -
-                double target =
-                    -(Kp * error + Ki * this->integral_error + Kd * derivative_error);  // PID commands to send to plant
+                double target = -(this->Kp * error + this->Ki * this->integral_error +
+                                  this->Kd * derivative_error);  // PID commands to send to plant
```

Comment:
going to assume this maths is still correct

Reply:
yep, just pointing it explicitly to the class variable (if in the future we have other variables named the same)